# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/grub2/hotfix-mkiamgexx.patch
# Copyright (C) 2026 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- grub-2.14/util/grub-mkimagexx.c.vanilla	2026-01-18 19:16:43.066726229 +0100
+++ grub-2.14/util/grub-mkimagexx.c	2026-01-18 19:17:24.142463016 +0100
@@ -2405,7 +2405,7 @@
 				kernel_path,
 				(unsigned long long) grub_host_to_target_addr (s->sh_addr),
 				(unsigned long long) image_target->link_addr);
-	    grub_util_error ("%s", msg);
+	    grub_util_warn ("%s", msg);
 	  }
       }
 
